rust_src/remacs-macros/lisp_attr.rs

```diff
@@ -39,13 +44,15 @@ fn parse_kv(kv_list: Vec<(syn::Ident, syn::StrLit)>, function: &Function) -> Lis
             "name" => name = Some(string.value),
             "c_name" => c_name = Some(string.value),
             "min" => min = i16::from_str(&string.value).unwrap(),
+            "intspec" => intspec = Some(string.value),
             _ => (), // TODO: throw a warning?
         }
     }
     LispFnArgs {
         name: name.unwrap_or_else(|| function.name.to_string().replace("_", "-")),
         c_name: c_name.unwrap_or_else(|| function.name.to_string()),
         min: min,
+        intspec: intspec,
```

Comment:
You can just say `intspec,` here (new feature from a few versions back)

Reply:
Updated.